### ISSUE TITLE
no such column: user.movieQuotaLimit

### ISSUE THREAD

**Issue by mindphaseproductions** [sct/overseerr#3594]
### Description

Unable to login to Overseerr with any Plex accounts in the last day or so. I receive an "Unable to authenticate." error. I didn't touch any settings. 

*Note: I replaced the IP in my logs that I pasted here with #s. Just FYI.

Please see logs, and let me know if any further details are required.

Thanks in advance!

### Version

1.33.2

### Steps to Reproduce

1. Attempt to Login with Plex account.
2. See that the login is denied, and "Unable to authenticate." is displayed.

### Logs

```shell
[error][API]: Something went wrong authenticating with Plex account {"errorMessage":"SQLITE_ERROR: no such column: user.movieQuotaLimit","ip":"::ffff:###.##.#.#"}
```

### Platform

desktop

### Device

PC

### Operating System

Linux Mint

### Browser

Chrome, Firefox

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by mindphaseproductions** [comment]
"movieQuota" columns were removed, jellyfin ones were added for some reason? Added them back and now all is good.

**Comment by vCybah** [comment]
@mindphaseproductions If you use the hotio image and have auto-updating then it's probably because of [this](https://imgur.com/CohR0Kk), it happened to me also and I guess I'm gonna have to restore from a backup from what everyone's saying unless you could explain how you fixed yours without restoring maybe? Seems like hotio accidentally turned his overseerr into jellyseerr or something lol Unless this is not what caused your issue as well, I never had this issue til now and I'm thinking this is why.  Overseerr has worked great for me otherwise, but I don't think it's actually an issue with Overseer. Im gonna try to look at a backup of the DB and see what's changed but I'm an idiot so if you could tell me how to fix it I'd really appreciate it, thanks.

**Comment by vCybah** [comment]
Okay yup just needed to add the missing stuff from the old DB to the new one and it's working again.
https://i.imgur.com/uIIT3YN.png
Old DB on the left, the new is one missing movieQuotaLimit plus the 3 below it so I just added all of them. Hopefully, nothing ends up getting borked by me doing that but as of right now, it's fixed for me as well.

**Comment by mindphaseproductions** [comment]
> Okay yup just needed to add the missing stuff from the old DB to the new one and it's working again. https://i.imgur.com/uIIT3YN.png Old DB on the left, the new is one missing movieQuotaLimit plus the 3 below it so I just added all of them. Hopefully, nothing ends up getting borked by me doing that but as of right now, it's fixed for me as well.

Yeah sorry, my reply was a little vague. I used a DB editor (DBeaver) to review the problematic .config/appdata/overseerr/db/db.sqlite3 file, saw there was those missing columns from your screenshot, and jellyfin ones in their place, so I copied the mentioned file from a backup from 2 days prior to this issue which had those 4 missing columns you showed in your screenshot from the proper db, and that fixed it. You didn't Bork anything up. I had a friend check his db, as he wasn't effected by the update, and it's the same as ours. Also, thanks for clearing up and finding out why this happened! I was wondering that myself. Much appreciated!
